apps/client/src/app/challenges/add/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { Anchor, Box, Breadcrumbs, Divider, Title } from '@mantine/core';
+
+import ChallengesForm from '@/components/form/ChallengesForm';
+
+const items = [
+  { title: 'Challenges', href: '/challenges' },
+  { title: 'Add Challenge', href: '#' }
+].map((item, index) => (
+  <Anchor href={item.href} key={index}>
+    {item.title}
+  </Anchor>
+));
+
+const AddChallenge = () => {
+  return (
+    <>
+      <Box component="section" p="2em" w={'100%'} mx="auto">
```

Comment:
I have created a component called `Container` that serves this purpose. You can use that here instead.

Reply:
Done